From ad5bd08f482f23c040491c6f698babf31bc5af93 Mon Sep 17 00:00:00 2001
From: Brian Norris <briannorris@chromium.org>
Date: Wed, 5 Feb 2020 15:10:01 -0800
Subject: [PATCH] iw: reg: parse the NO_HE regulatory flag

This isn't supported in wireless-regdb yet, but self-managed drivers are
preparing to report it:

https://patchwork.kernel.org/patch/11343173/
https://lkml.kernel.org/linux-wireless/20200121081213.733757-1-luca@coelho.fi

Signed-off-by: Brian Norris <briannorris@chromium.org>
Link: https://lore.kernel.org/r/20200205231001.132595-3-briannorris@chromium.org
Signed-off-by: Johannes Berg <johannes.berg@intel.com>
---
commit ad5bd08f482f23c040491c6f698babf31bc5af93 upstream

 reg.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/reg.c b/reg.c
index a2368df39009..db1b12906519 100644
--- a/reg.c
+++ b/reg.c
@@ -209,6 +209,7 @@ static int print_reg_handler(struct nl_msg *msg, void *arg)
 		PARSE_FLAG(NL80211_RRF_NO_HT40PLUS, "NO-HT40PLUS");
 		PARSE_FLAG(NL80211_RRF_NO_80MHZ, "NO-80MHZ");
 		PARSE_FLAG(NL80211_RRF_NO_160MHZ, "NO-160MHZ");
+		PARSE_FLAG(NL80211_RRF_NO_HE, "NO-HE");
 
 		/* Kernels that support NO_IR always turn on both flags */
 		if ((flags & NL80211_RRF_NO_IR) && (flags & __NL80211_RRF_NO_IBSS)) {
-- 
2.26.0.292.g33ef6b2f38-goog

